Tests/Tests_REST_API/TestData.py

```diff
@@ -0,0 +1,21 @@
+from random import randint
+
+
+class DataTest:
+    customerId = 12656
+    amount = 5000
+    new_account_id = randint(10000, 99999)
+    newAccountType = 0
+    username = 'ivan'
+    password = 'ivan123'
+
+
+class DataEndPoints:
+    base_uri = 'https://parabank.parasoft.com/parabank/services/bank'
```

Comment:
Don't put data like DataTest.customerId inside endpoint, add this data when you create test

Reply:
Done